qiita_pet/handlers/api_proxy/studies.py

```diff
@@ -0,0 +1,71 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+from __future__ import division
+
+from qiita_db.study import Study
+from qiita_pet.handlers.api_proxy.util import check_access
+
+
+def data_types_get_req():
+    """Returns data types available in the system
+
+    Returns
+    -------
+    list of str
+        Data types available on the system
+    """
+    return {'status': 'success',
+            'message': '',
+            'data_types': Study.all_data_types()
```

Comment:
I just saw this and I think this is not correct. Why should we go through the study to retrieve all the data types? Shouldn't it be a util function? This function has been merged in the artifact-study-pages branch previously but I highly recommend removing it from the study object and create a new util function.